types/webpack-bundle-analyzer/tsconfig.json

```diff
@@ -14,10 +14,10 @@
         ],
         "paths": {
```

Comment:
Are path mappings still necessary?

Reply:
I'm not sure, I'd copied them from #52044. 
When I remove them and run `npm test webpack-bundle-analyser` I get the following output
```
Error: Errors in typescript@4.3 for external dependencies:
../webpack/index.d.ts(30,1): error TS2303: Circular definition of import aluffer'. Do you need to install type definitions for node? Try `npm i --save-deias 'webpack'.
```
It looks like without these paths the test code is trying to use the typings in this repo and getting confused. I'm not familiar enough with all this stuff to fully understand why this is happening. Is the correct thing to to here to follow the [Removing a package](https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package) for the @types/webpack package for v5+ and tapable v2+ seeing as webpack and tapable now bundle their own types?
```

Reply:
Looking through the removing a package instructions maybe that's not been done here deliberately to keep backward compatibility with older typings of webpack where needed and looking at the v5 typings for webpack in this repo I am beginning to understand the circular reference error. Is there a special section of the readme I'm missing for these cases where the typings are now part of the package but not removed from here for backward compatibility reasons?

Reply:
Huh, I don't know what it means for webpack to require itself.  @sandersn?

Reply:
@ksstott is basically right. @types/webpack@4 needs to stay on DT to keep receiving updates, but updated plugins like this one need to be able to refer to webpack@5, which ships its own types.

The solution is for updated plugins to add package.json like you'd expect (so that the plugin has the correct dependencies when installed), but also use path mappings so that the DT tests resolve to a local copy of webpack@5. The tsconfig doesn't ship to npm, so the path mapping is just to make DT compile correctly.

This is a one-off for webpack for now, and I haven't figured out how or where to document it.